### ISSUE TITLE
Some attributes do not work with vecty.Property()

### ISSUE THREAD

**Issue by glight2000** [gopherjs/vecty#56]
Keys does not work:
`aria-hidden`
`role`
May be it's good to support a method like `setAttributs(key, value)` to handle customize attribute names.

**Comment by slimsag** [comment]
Looks like there are certain attributes (like `aria-hidden`) which cannot be set using a DOM property.

I think we will need something akin to `vecty.Property` but for attributes (`vecty.Attribute`) to handle this case.

**Comment by slimsag** [comment]
I've added this in 34ae770

In an ideal world we would also probably have an `attr` package and limit the scope of `Attribute` to just a few things that do not have a property, but that will be very involved I think (to determine which ones). Let's address that in the future once we get a better idea for which attributes people use in the wild.
